isthmus/src/main/java/io/substrait/isthmus/SqlExpressionToSubstrait.java

```diff
@@ -151,9 +151,6 @@ private Result registerCreateTablesForExtendedExpression(List<String> tables)
           }
         }
       }
-    } else {
```

Comment:
> What is the reason for removing this? Is it because of the if/else handle?

The main reason for this is that for literal expressions `./isthmus -e "1"` no table/column/type definition is required.